gatsby-config.js

```diff
@@ -108,5 +108,48 @@ module.exports = {
     },
     'gatsby-plugin-purgecss', // must be after other CSS plugins
     'gatsby-plugin-netlify', // make sure to keep it last in the array
+    {
+      resolve: `gatsby-transformer-remark`,
```

Comment:
Per the comment above ` // make sure to keep it last in the array` we should probably re-order